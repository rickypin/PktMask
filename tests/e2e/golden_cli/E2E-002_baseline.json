{
  "test_id": "E2E-002",
  "name": "Anonymize Only",
  "input_file": "tests/data/tls/tls_1_2-2.pcap",
  "input_hash": "baf39e40187111119c0de19358b3c944a07489806231fd24e6ab922b678b3ded",
  "cli_args": [
    "--anon"
  ],
  "output_hash": "4871caf1e13175a7de0dfdaa5fe433f595145c2f5eb6acf954d245ee98c201ed",
  "timestamp": "2025-10-09T21:31:08.144597",
  "version": "1.0-cli",
  "note": "Generated using CLI only - completely decoupled from API"
}