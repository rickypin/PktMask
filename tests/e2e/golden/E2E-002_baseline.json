{
  "test_id": "E2E-002",
  "name": "Anonymize Only",
  "input_file": "tests/data/tls/tls_1_2-2.pcap",
  "input_hash": "baf39e40187111119c0de19358b3c944a07489806231fd24e6ab922b678b3ded",
  "config": {
    "dedup": false,
    "anon": true,
    "mask": false
  },
  "output_hash": "4871caf1e13175a7de0dfdaa5fe433f595145c2f5eb6acf954d245ee98c201ed",
  "output_file_size": 2721,
  "packet_count": 14,
  "stats": {
    "packets_processed": 14,
    "packets_modified": 14,
    "duration_ms": 5.303144454956055,
    "stages": [
      {
        "name": "AnonymizationStage",
        "packets_processed": 14,
        "packets_modified": 14,
        "duration_ms": 5.232095718383789
      }
    ]
  },
  "timestamp": "2025-10-09T19:39:44.742426",
  "version": "1.0"
}